components/ConfirmDelete/index.js

```diff
@@ -12,10 +15,12 @@ export default function ConfirmDelete({ handleDeleteTrip }) {
   }
   return (
     <div>
-      <p>Are you sure ?</p>
-      <button type="button" onClick={handleDeleteTrip}>
-        Yes
-      </button>
+      <p>Are you sure?</p>
+      <Link href="/">
+        <button type="button" onClick={handleDeleteTrip}>
+          Yes
+        </button>
+      </Link>
```

Comment:
That's clever!

But it's abusing the infrastructure a bit. In order to redirect to the homepage, use the router inside your `handleDeleteTrip` function, like:

```js
import { useRouter } from 'next/router';

...

const router = useRouter();

function handleDeleteTrip() {
  // what you need to do to delete the trip
  router.push('/');
}
```